docs/tools/ocaml.md

```diff
@@ -12,7 +12,7 @@ Then, install the dependencies with `opam install core yojson`.
 Use
 ---
 
-You can include the library by adding it to the `libraries` substanza of your `dune` file.
+You can include the library by running `opam pin add bril https://github.com/sampsyo/bril/tree/master/bril-ocaml`. You can then include it in your dune files as `bril`, like any other library!
```

Comment:
Maybe this would be more legible as a standalone code block, perhaps? Like this:

```
…by running:

    opam pin add bril https://github.com/sampsyo/bril/tree/master/bril-ocaml

You can then…
```